internal/db/types.go

```diff
@@ -24,8 +24,31 @@ type ChangeRequest struct {
 }
 
 type Service struct {
-	Id         int
-	ResourceId int
+	Id                     int
+	CreatedAt              sql.NullTime
+	UpdatedAt              sql.NullTime
```

Comment:
I'm not sure if this Go sql package has a non-nullable Time type, but according to the [schema.rb](https://github.com/ShelterTechSF/askdarcel-api/blob/8559a6e704c604c3fb1285a299ccacc03a706fd6/db/schema.rb#L382-L383) file in the Rails app, created_at and updated_at are not nullable